dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -1372,9 +1372,11 @@ void
 SubModel::printInitValuesVariables(std::ofstream& fstream) {
   fstream << " ====== INIT VARIABLES VALUES ======\n";
   const vector<string>& xNames = xNamesInit();
-  for (unsigned int i = 0; i < yLocalInit_.size(); ++i)
-    fstream << std::setw(50) << std::left << xNames[i] << std::right << ": y =" << std::setw(15) << DYN::double2String(yLocalInit_[i])
-      << " yp =" << std::setw(15) << DYN::double2String(ypLocalInit_[i]) << "\n";
+  if (!xNames.empty()) {
```

Comment:
When do we have yLocalInit_.size() != xNames.size()?

Reply:
I think ti was more an issue that the vector was not initialized is some scenario